code/datums/elements/food/microwavable.dm

```diff
@@ -6,14 +6,18 @@
 	var/atom/default_typepath = /obj/item/food/badrecipe
 	/// Resulting atom typepath on a completed microwave.
 	var/atom/result_typepath
+	/// Reagents that should be added to the result
+	var/list/added_reagents
 
-/datum/element/microwavable/Attach(datum/target, microwave_type)
+/datum/element/microwavable/Attach(datum/target, microwave_type, list/reagents = list())
```

Comment:
Because this is an element, you need to pass a static cached list to it.

Reply:
Would changing the declaration for `added_reagents` to `var/list/added_reagents = list()` and then removing the default argument work?

Reply:
No, any place that adds `/datum/element/microwavable` with reagents needs to do `var/static/list/microwave_reagents = list(blah = 1)`.

Reply:
To explain more, bespoke elements know when to re-use an element by checking the arguments you pass into it. 

Lists are objects, so while two identical lists may look identical they have different references, so it makes two "identical" bespoke elements. 

So to get around that we pass static lists into bespoke elements, these static lists are the same object / same list across all types so it passes the exact same list in (and we get the same element as expected).

Reply:
I've made it use static lists now. All the unit tests passed, and all the donks appeared to have the right chems when I tested it in game, so I think I did it right.